mysql-compatibility.md

```diff
@@ -93,171 +76,118 @@ mysql> select _tidb_rowid, id from t;
 
 ### Performance schema
 
-Performance schema tables return empty results in TiDB. TiDB uses a combination of [Prometheus and Grafana](/monitor-a-tidb-cluster.md) for performance metrics instead.
-
-TiDB supports the `events_statements_summary_by_digest` table from TiDB 3.0.4. For more information, see [Statement Summary Table](/statement-summary-tables.md).
+TiDB uses a combination of Prometheus and Grafana to store and query the performance monitoring metrics. The Performance schema is an empty schema.
```

Comment:
```suggestion
TiDB uses a combination of Prometheus and Grafana to store and query the performance monitoring metrics. Some performance schema tables return empty results in TiDB.
```

Reply:
Looks like I didn't understand this sentence...